containers/Ideas/IdeasList.js

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
-import { View } from 'react-native';
+import {View} from 'react-native';
 
-import { IdeasListItem } from './IdeasListItem';
+import {IdeasListItem} from './IdeasListItem';
+import {TextSourceSans} from '../../components/TextSourceSans';
 
-export const IdeasList = (props) => {
+export const IdeasList = props => {
   //do not pass ideas to child components
   const {ideas, ...otherProps} = props;
+  const hasIdeas = !!ideas && ideas.length > 0;
   return (
     <View>
-      {props.ideas.map((idea) => {
-        return (
-          <IdeasListItem
-            key={`pk${idea.pk}`}
-            idea={idea}
-            {...otherProps}
-          />
-        );
-      })}
+      {hasIdeas ? (
+        props.ideas.map(idea => {
+          return (
+            <IdeasListItem key={`pk${idea.pk}`} idea={idea} {...otherProps} />
+          );
+        })
+      ) : (
+        <TextSourceSans>
+          Could not fetch ideas. Do you use the correct Backend and is your
+          local adhocracy+ server running?
+        </TextSourceSans>
```

Comment:
should we also add text for if there are just no ideas like we do on a+ with 'Nothing to show'?

Reply:
hehe, yes i did it more user oriented now.